ui/src/components/EventsCarousel/index.tsx

```diff
@@ -2,63 +2,81 @@ import styles from './styles.module.scss';
 import Carousel from 'react-multi-carousel';
 import 'react-multi-carousel/lib/styles.css';
 import {dummyNewsImg} from '../../assets';
+import {HeadingDropdown} from '../HeadingDropdown';
+import {useState} from 'react';
+import {
+  EVENT_OPTIONS,
+  RESPONSIVE_BREAKPOINTS_CAROUSEL,
+} from '../../globals/constants';
 
 const data = [
   {
     name: 'Aperta Fons',
     club: 'TPC',
     imgUrl: dummyNewsImg,
+    startTime: new Date(2020, 0),
+    endTime: new Date(2020, 0),
   },
   {
     name: 'Aperta Fons',
     club: 'TPC',
     imgUrl: dummyNewsImg,
+    startTime: new Date(2020, 0),
+    endTime: new Date(2024, 0),
   },
   {
     name: 'Aperta Fons',
     club: 'TPC',
     imgUrl: dummyNewsImg,
-  },
-  {
-    name: 'Aperta Fons',
-    club: 'TPC',
-    imgUrl: dummyNewsImg,
-  },
-  {
-    name: 'Aperta Fons',
-    club: 'TPC',
-    imgUrl: dummyNewsImg,
+    startTime: new Date(2023, 0),
+    endTime: new Date(2024, 0),
   },
 ];
 
+interface EventsCarouselData {
+  name: string;
+  club: string;
+  imgUrl: string;
+  startTime: Date;
+  endTime: Date;
+}
+
+export const shouldRenderEvent = (val: string, event: EventsCarouselData) => {
```

Comment:
When reading the method signature, I don't understand what `val` is.

maybe `eventType`? Can it be an enum?